{
  "modified": "2024-12-09T06:49:53Z",
  "published": "2024-12-09T06:49:53Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-11542",
  "summary": "Malicious code in byte-flux-3822a6dd00d6414daba3ae0de3930a5a (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "byte-flux-3822a6dd00d6414daba3ae0de3930a5a",
        "purl": "pkg:pypi/byte-flux-3822a6dd00d6414daba3ae0de3930a5a"
      },
      "versions": [
        "9.6"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-10988",
        "import_time": "2024-12-09T14:38:41.547572343Z",
        "modified_time": "2024-12-09T06:49:53Z",
        "sha256": "654071e046a5b7fdecba02c3c3cbf5c243a254e7abf1143e45c9c17f80f0eb73",
        "source": "reversing-labs",
        "versions": [
          "9.6"
        ]
      }
    ]
  }
}
